Cake.ArgumentHelpers/Cake.ArgumentHelpers.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<AssemblyName>Cake.ArgumentHelpers</AssemblyName>
-		<TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+		<TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```

```suggestion
		<TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
```

Looks good. Let's just sort it from newer to older so that `net6.0` is the default when opening in VS

---

Good idea.